Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -1153,36 +1339,102 @@ private string CreateNewTestCase(ActivitiesGroup activitiesGroup, string fatherI
             return test.Id.ToString();
         }
 
-        private string UpdateTestCase( ActivitiesGroup activitiesGroup, string fatherId, ObservableList<ExternalItemFieldBase> testCaseFields, string testScript)
+        private void AddEntityFieldValues(ObservableList<ExternalItemFieldBase> fields, BaseEntity test, string entityType)
+        {
+            foreach (ExternalItemFieldBase field in fields)
+            {
+                try
+                {
+                    if ((field.ToUpdate || field.Mandatory) && !test.Contains(field.ExternalID))
+                    {
+                        if (string.IsNullOrEmpty(field.SelectedValue) == false && field.SelectedValue != "Unassigned")
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6361231)